test/functional/services/combo_box.ts

```diff
@@ -281,6 +281,17 @@ export function ComboBoxProvider({ getService, getPageObjects }: FtrProviderCont
         .map(option => $(option).text());
       return selectedOptions.length === 1 && selectedOptions[0] === value;
     }
+
+    /**
+     * Clears input field
+     * @param comboBoxSelector data-test-subj selector
+     */
+    public async clearInputField(comboBoxSelector: string): Promise<void> {
+      log.debug(`comboBox.clear, comboBoxSelector:${comboBoxSelector}`);
+      const comboBoxElement = await testSubjects.find(comboBoxSelector);
+      const input = await comboBoxElement.findByTagName('input');
+      await input.clearValueWithKeyboard();
+    }
   }
```

Comment:
input field of index pattern combobox on TSVB panel is not cleared with [existing clearing function](https://github.com/elastic/kibana/blob/master/test/functional/services/combo_box.ts#L143), but since it is TSVB only I added a separate method

Reply:
Great, thanks @dmlemeshko !